oas

```diff
@@ -1,506 +1,354 @@
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/Error"
-                }
-              }
-            }
-          },
-          "500": {
-            "description": "A generic error message, given when an unexpected condition was encountered and no more specific message is suitable.",
-            "content": {
-              "application/json": {
-                "schema": {
-                  "$ref": "#/components/schemas/Error"
-                }
-              }
-            }
-          }
-        }
-      }
-    }
-  },
-  "components": {
-    "securitySchemes": {
-      "bearerAuth": {
-        "type": "http",
-        "scheme": "bearer",
-        "bearerFormat": "JWT"
-      }
-    },
-    "schemas": {
-      "ParameterTypes": {
-        "type": "string",
-        "enum": [
-          "ATMO__T",
-          "ATMO__RH",
-          "ATMO__P",
-          "ATMO__WIND_SPEED",
-          "ATMO__IRRADIATION",
-          "ATMO__SSR",
-          "ATMO__WIND_GUSTINESS",
-          "ATMO__RAINFLOW",
-          "ATMO__RAINFALL",
-          "SOIL__VMC",
-          "SOIL__T",
-          "CANOPY__WETNESS",
-          "SOIL__WETNESS"
-        ]
-      },
-      "AggregationTypes": {
-        "type": "string",
-        "enum": [
-          "min",
-          "max",
-          "avg",
-          "sum"
-        ]
-      },
-      "DataSeriesType": {
-        "description": "A data series of nullable decimal numbers.",
-        "example": [
-          -12.4,
-          null,
-          4,
-          5.6
-        ],
-        "type": "array",
-        "items": {
-          "anyOf": [
-            {
-              "type": "number"
+    "components": {
+        "schemas": {
+            "AssetDataRequest": {
+                "title": "AssetDataRequest",
+                "required": [
+                    "identifier"
+                ],
+                "type": "object",
+                "properties": {
+                    "identifier": {
+                        "type": "string",
+                        "description": "Physical identifier (MAC address) of the asset. Required*",
+                        "example": "12:34:56:78:90:AB"
+                    },
+                    "from": {
+                        "type": "string",
+                        "description": "The timestamp to start from. Optional",
+                        "format": "date-time",
+                        "example": "2022-10-10T12:47:05.899139"
+                    },
+                    "to": {
+                        "type": "string",
+                        "description": "The timestamp to end at. Optional",
+                        "format": "date-time",
+                        "example": "2022-10-11T12:47:05.899152"
+                    },
+                    "limit": {
+                        "type": "integer",
+                        "description": "Total number of data points to be returned. Optional",
```

Comment:
Don't use "Optional" in text. Make use of OAS atttributes that identify required attributes (and optional by default).
Explain what the default for "limit" is if not specified